### ISSUE TITLE
Buttons for copy, cut, paste, undo.

### ISSUE THREAD

**Issue by cedarbob** [fabiospampinato/vscode-commands#26]
I just installed the extension and tried to add buttons for cut, copy, paste and undo with mixed results.  The buttons for cut and undo worked just fine.  But the buttons for copy and paste didn't work.  I apologize in advance if I missed the obvious.   Here is my commands.json:

{
  "commands": [
    {
      "command": "commands.refresh",
      "text": "$(sync)",
      "tooltip": "Refresh commands",
      "color": "#FFCC00"
    },
    {
      "text": "Copy",
      "command": "copy",
      "tooltip": "Copy selection."
    },
    {
      "text": "Cut",
      "command": "cut",
      "tooltip": "Cut selection."
    },
    {
      "text": "Paste",
      "command": "paste",
      "tooltip": "Copy selection."
    },
    {
      "text": "Undo",
      "command": "undo",
      "tooltip": "Undo."
    }
  ]
}

**Comment by fabiospampinato** [comment]
Where did you get those commands names?

The commands according to the shortcuts file should be: execCopy, execCut and execPaste.

**Comment by cedarbob** [comment]
I tried using those command names instead of copy, cut and paste - and none worked.  I got the "Command xyz not found" message for each.  I tried this on both the Windows and Linux versions of VSC.  I also tried prepending various qualifiers to copy and paste and couldn't crack the code.  If anyone has a clue which command names should be used to get copy and paste to work with this extension I'd like to know.
